src/components/PageAccess/NoAccessState.tsx

```diff
@@ -40,6 +40,7 @@ const NoAccessState: React.FC<NoAccessStateProps> = ({
       variant={EmptyStateVariant.full}
       {...props}
       data-test="no-access-state"
+      data-testid="no-access-state"
```

Comment:
Ideally we should switch to only using `data-test-id` but some tests still use `data-test` and to support that we need to run `configure({ testIdAttribute: 'data-test' })` for every test file.
I wanted to update all of those but that's too many unrelated changes to this feature, so I added this change so new test files don't have to run configure until we can clean the old ones.

I guess we can skip it for now since this is not relevant to this PR.